percona/controllers/deployment/deployment.go

```diff
@@ -51,3 +51,15 @@ func UpdateDeploymentImage(deployment *appsv1.Deployment, image string) {
 
 	deployment.Spec.Template.Spec.Containers = containers
 }
+
+func UpdateDeploymentPGBadgerImage(deployment *appsv1.Deployment, image string) {
```

Comment:
Why do we need to add this new fun? Maybe we can extend and use  this one UpdateDeploymentImage ?

Reply:
https://github.com/percona/percona-postgresql-operator/pull/152/commits/8b77881ffe9d399948d533623ba36fe868930673